packages/backend/src/Application.ts

```diff
@@ -30,12 +31,31 @@ export class Application {
     )
 
     const metrics = new Metrics()
+    const repositoryHistogram: RepositoryHistogram = metrics.createHistogram({
```

Comment:
Move this inside Metrics constructor and expose something like `getRepositoryHistogram`.